google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ClientSideStatementValueConverters.java

```diff
@@ -342,4 +342,17 @@ public Priority convert(String value) {
       return values.get("PRIORITY_" + value);
     }
   }
+
+  static class ExplainCommandConverter implements ClientSideStatementValueConverter<String> {
+
+    @Override
+    public Class<String> getParameterClass() {
+      return String.class;
+    }
+
+    @Override
+    public String convert(String value) {
+      return value.split(" +", 2)[1].toLowerCase();
```

Comment:
A couple of points regarding this:
1. I think it should be programmed a little more defensively, so check that the array actually contains at least 2 items before trying to get the element at index 1, so we know that we won't run into any `ArrayIndexOutOfBounds` exceptions.
2. It should probably be `value.split("\\s+")`. That ensures that the statement is also correctly handled if there is a line feed or a tab between `EXPLAIN` and the rest of the statement. Please also add tests for that.
3. Why the `.toLowerCase()`? I don't think we need (or should) call that on the entire sql string. If we need it to determine whether the first token in the SQL string is one of the `explain` options, then we should only call `toLowerCase()` for that part. Otherwise, we will be sending a modified SQL string to the backend.